### ISSUE TITLE
Removing forceful aliases

### ISSUE THREAD

**Issue by OssiPesonen** [OssiPesonen/Elixir#1]
Currently the library forces alias use for joins. 

```php
$qb->select('parent_alias.id', 'alias_for_joined_table.name')
->from('parent', 'parent_alias')
->innerJoin('parent_alias', 'joined_table', 'alias_for_joined_table', 'alias_for_joined_table.parent_id = parent_alias.id');
```

This creates an SQL query:

```sql
SELECT parent_alias.id, alias_for_joined_table.name FROM parent parent_alias INNER JOIN joined_table alias_for_joined_table ON alias_for_joined_table.parent_id = parent_alias.id
```

This becomes cumbersome as it's sometimes easier to build a query without the need to create an alias for every table, but instead reference the table name. 

The code should ignore alias use in cases where the alias is passed as:

- Real name
- Empty string
- NULL value
- Omitted (from)

In these cases aliases should be dropped. Example:

```php
$qb->select('parent_alias.id', 'alias_for_joined_table.name')
->from('parent')
->innerJoin('parent', 'joined_table', null, 'joined_table.parent_id = parent.id');
```

Resulting in:

```sql
SELECT parent.id, joined_table.name FROM parent INNER JOIN joined_table ON joined_table.parent_id = parent.id
```

**Comment by OssiPesonen** [comment]
Documentation added https://rcls.gitbook.io/elixir/building-a-query/aliases#exclude-aliases

Released in [0.1.7](https://github.com/OssiPesonen/Elixir/releases/tag/0.1.7)

**Comment by OssiPesonen** [comment]
Copy from the Release notes:

> This will potentially change any previously built SQL clauses without breaking them, if you've passed the table name to $alias creating a duplicate.

```php
$queryBuilder
    ->select('parent.id')
    ->from('parent')
    ->join('parent', 'child', 'child', 'child.fk_parent = parent.id')

// SELECT parent.id FROM parent INNER JOIN child child ON child.fk_parent = parent.id
```

> This` will now remove the second child from the SQL.
